lib/config/options/index.ts

```diff
@@ -2844,6 +2844,13 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'milestone',
+    description: `The number of a milestone. If set, the milestone will be set when Renovate creates the PR.`,
+    type: 'integer',
```

Comment:
Why this was changed to `integer` this would make it probably hard to implemet on other platforms which don't use int as primary key.

it's also much harder to remember the id instead of the title-